sections/featured-product-slider.liquid

```diff
@@ -58,6 +58,10 @@
   .section-featured-products-{{ id }}.product-slider.splide .splide__arrow svg {
     fill: {{ section.settings.slider_arrows_color.hex }};
   }
+
+  .section-featured-products-{{ id }} .container .counter-wrapper .countdown-wrapper .message {
+    text-align: center;
+  }
```

Comment:
I think we can move this to css since the styling is not dependant on a parameter

Reply:
That's how it was before but Adil reported that the countdown is used in the products page and there, the message is right aligned